app/controller.js

```diff
@@ -21,4 +21,20 @@ exports.getEndpoints = (req,res,next)=>{
 
         return res.status(200).send({endpoints:parsedEndpoints})
     })
+}
+
+exports.getArticles = (req,res,next) =>{
+
+    const articleId = req.params.article_id
+
+    selectAllArticles(articleId).then(({rows})=>{
+        if(rows.length===0){return res.status(400).send({msg:"Invalid query params!!"})}
+        
+        return res.status(200).send({article:rows[0]})
+    
+    }).catch(({code})=>{
+        if(code === "22P02"){
```

Comment:
setting up an error handling chain will help us handle the sending of all of these errors